cradle-cassandra/src/main/java/com/exactpro/cradle/cassandra/CassandraCradleStorage.java

```diff
@@ -321,6 +373,10 @@ protected void doStoreGroupedMessageBatch(GroupedMessageBatchToStore batch, Page
 					messagesWorker.storeSession(b).get();
 					messagesWorker.storePageSession(b, pageId).get();
```

Comment:
Why can we just reuse these methods?

Reply:
New code use Grouped message batch instead of creating and calculating MessageBatch